pkg/labelsfilter/filter.go

```diff
@@ -131,9 +148,24 @@ func ParseLabelPrefixCfg(prefixes []string, file string) error {
 		cfg.LabelPrefixes = append(cfg.LabelPrefixes, p)
 	}
 
+	if fromCustomFile {
+		found := false
+		for _, label := range cfg.LabelPrefixes {
+			if label.Source+":"+label.Prefix == reservedLabelsPattern {
+				found = true
+				break
+			}
+		}
+
+		if !found {
+			log.Errorf("'%s' needs to be included in the final label list for " +
```

Comment:
The plus should be right after the double quotes here. See the GitHub action failures. Sorry for misleading you :disappointed: